File: host_virtual_M/index/send_email.py
import smtplib
from email.message import EmailMessage

# import os
# file_path = 'C:/xampp/htdocs/sistema_de_tickets/sistema_de_tickets/host_virtual_M/index/formDataUser.txt'
# os.chdir('C:/xampp/htdocs/sistema_de_tickets/sistema_de_tickets/host_virtual_M/index')  # Cambia el directorio de trabajo a la carpeta del archivo
# print(os.path.exists(file_path))  # Imprime True si el archivo existe, False si no.    

# Función para leer datos del archivo en UTF-8
def read_form_data(file_path):
    data = {}
    with open(file_path, 'r', encoding='utf-8') as file:
        for line in file:
            key, value = line.strip().split('=', 1)
            data[key] = value
    return data

# Ruta al archivo que contiene los datos del formulario
file_path = 'C:/xampp/htdocs/sistema_de_tickets/sistema_de_tickets/host_virtual_M/index/form_data_user.txt'

# Leer datos del archivo
form_data = read_form_data(file_path)

# Configuración del correo para Gmail
email_subject = "Confirmación de recepción del ticket"
sender_email_address = "sistemadiprop@outlook.com"
email_password = "vvvglnmafyngyqlz"  
email_smtp = "smtp.office365.com"
smtp_port = 587

# Crear el mensaje de correo
message = EmailMessage()
message['Subject'] = email_subject
message['From'] = sender_email_address
message['To'] = f"{form_data.get('correo')}, cecilio@iplgsc.com"  # Usar el correo del formulario
message.set_content(
    f"Hola {form_data.get('nombrecompleto', 'Usuario')},\n\n"
    f"Gracias por contactarnos. Aquí aparecen los datos que nos suministró para confirmar su correcto envío:\n\n"
    f"Nombre Completo: {form_data.get('nombrecompleto', 'No proporcionado')}.\n"
    f"Descripción: {form_data.get('descripcion', 'No proporcionado')}\n"
    f"Departamento: {form_data.get('ubicacion', 'No proporcionado')}.\n"
    f"Urgencia: {form_data.get('urgencia', 'No proporcionado')}.\n\n"
    "Atentamente,\nEl departamento de Mantenimiento\n(no responder a este mensaje).",
    charset='utf-8'
)

try:
    # Conectar al servidor SMTP de Gmail
    with smtplib.SMTP(email_smtp, smtp_port) as server:
        server.ehlo()  # Saludo inicial al servidor
        server.starttls()  # Comenzar la conexión segura TLS
        server.ehlo()  # Repetir saludo después de TLS
        server.login(sender_email_address, email_password)  # Iniciar sesión en Gmail
        server.send_message(message)  # Enviar el correo

    print("Correo enviado exitosamente.")  # Mensaje de éxito

except Exception as e:
    print(f"Error al enviar el correo: {e}")  # Mensaje de error con detalles

File: host_virtual_TI/index/send_email.php
<?php
// Aquí incluyes la configuración de PHPMailer y los datos de envío
use PHPMailer\PHPMailer\PHPMailer;
use PHPMailer\PHPMailer\Exception;

require 'PHPMailer/src/Exception.php';
require 'PHPMailer/src/PHPMailer.php';
require 'PHPMailer/src/SMTP.php';

$mail = new PHPMailer(true);

try {
    // Configuración del servidor SMTP de Gmail
    $mail->isSMTP();
    $mail->Host = 'smtp.gmail.com';
    $mail->SMTPAuth = true;
    $mail->Username = 'tickesPrueba1@gmail.com';
    $mail->Password = 'nfzs zcii xrhr hyky';
    $mail->SMTPSecure = PHPMailer::ENCRYPTION_STARTTLS;
    $mail->Port = 587;

    // Configuración del correo
    $form_data = json_decode(file_get_contents('form_data_user.json'), true);
    $mail->setFrom('tickesPrueba1@gmail.com', 'Departamento TI');
    $mail->addAddress($form_data['correo']);
    $mail->addAddress('ricaurte@iplgsc.com'); // Correo adicional

    $mail->isHTML(false); // Enviar el mensaje en formato texto
    $mail->Subject = 'Confirmación de recepción del ticket';
    $mail->Body    = "Hola " . $form_data['nombrecompleto'] . ",\n\n" .
                     "Gracias por contactarnos. Aquí están los datos que nos suministraste para confirmar su correcto envío:\n\n" .
                     "Nombre Completo: " . $form_data['nombrecompleto'] . ".\n" .
                     "Descripción: " . $form_data['descripcion'] . "\n" .
                     "Ubicación: " . $form_data['ubicacion'] . ".\n" .
                     "Urgencia: " . $form_data['urgencia'] . ".\n\n" .
                     "Atentamente,\nEl departamento de TI\n(no responder a este mensaje).";

    $mail->send();

    // Redirigir de vuelta a crear_ti.php con un mensaje de éxito
    header('Location: crear_ti.php?status=success');
    exit();
} catch (Exception $e) {
    // Redirigir de vuelta a crear_ti.php con un mensaje de error
    header('Location: crear_ti.php?status=error&message=' . urlencode($mail->ErrorInfo));
    exit();
}

?>

<!-- . ", ricaurte@iplgsc.com" -->